diff --git a/js/webform.js b/js/webform.js
index e33a28c..e3dde8c 100644
--- a/js/webform.js
+++ b/js/webform.js
@@ -159,9 +159,15 @@ Drupal.webform.conditionalCheck = function(e) {
      var $target = $form.find('.' + ruleGroup['target']);
      var $targetElements;
      if (showComponent) {
-        $targetElements = $target.find('.webform-conditional-disabled').removeClass('webform-conditional-disabled');
-        $.fn.prop ? $targetElements.prop('disabled', false) : $targetElements.removeAttr('disabled');
-        $target.show();
+        var elements = $target.find('.webform-conditional-disabled');
+        elements.each( function() {
+          if (this.spireEnable) {
+            this.spireEnable();
+          }
+        });
+        elements.removeAttr('disabled').removeClass('webform-conditional-disabled').end().show();
+
+
       }
       else {
         $targetElements = $target.find(':input').addClass('webform-conditional-disabled');;
